lib/rack/utf8_sanitizer.rb

```diff
@@ -125,7 +125,10 @@ def sanitize_rack_input(env)
       uri_encoded = URI_ENCODED_CONTENT_TYPES.any? {|type| content_type == type}
 
       if env['rack.input']
-        sanitized_input = sanitize_io(env['rack.input'], uri_encoded)
+        content_length = if env['CONTENT_LENGTH']
+          env['CONTENT_LENGTH'].to_i
+        end
+        sanitized_input = sanitize_io(env['rack.input'], uri_encoded, content_length)
```

Comment:
```suggestion
        sanitized_input = sanitize_io(env['rack.input'], uri_encoded: uri_encoded, content_length: env['CONTENT_LENGTH'].to_i)
```

Reply:
No we can't do that as `CONTENT_LENGTH` may not be present (chunked), so `nil.to_i` would return `0`

Reply:
Yeah, but the gemspec claim to support 1.9.3 (thought CI only test 2.5+), and safe navigation was only added in 2.3. So I can't use it.

Reply:
We can drop everything before 2.3.

Reply:
> We can drop everything before 2.3.

Done.